src/layer/Layer.Label.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Adds label-related methods to all layers.
+ */
+
+L.Layer.include({
+
+	bindLabel: function (content, options) {
```

Comment:
@mourner any good idea to factorize with L.Layer.bindPopup (same with openLabel/Popup, toggleLabel/Popup…)?

Mainly, we would need to:
- make `this._popup`/`this._label` dynamic
- make `instanceof L.Popup` dynamic also
- deal with some specific behaviour here and there (like calling `this._map._openLabel` or `this._map._openPopup` or label `clickable` option)
Plus we want to keep `marker.bindPopup` and `marker.bindLabel` with API just as simple as now.

I've thought about something like having `_bindPopupBase(content, options, type, class)` that would then be called by both `bindLabel` and `bindPopup` (eg. `this._bindPopupBase(content, options, 'label', L.Label)`). But that turns out to be quite ugly code, not very elegant nor easily readable.
Another option would be to have a proxy model (like we have `L.Marker.Drag`), but I feel this would be overcomplicating the internal API.

Thoughts? :)